commands/summary.js

```diff
@@ -0,0 +1,87 @@
+import * as bungie from '../bungieApi';
+import * as util from 'util';
```

Comment:
ah, ok - I was unfamiliar with the syntax, so I just copied the bungie one - I'll update them all, but I think I was only using util.format anyway... I'll check....